lib/utils.pm

```diff
@@ -420,9 +420,9 @@ sub sle_version_at_least {
           && !check_var($version_variable, '12-SP3');
     }
 
-    if ($version eq '13') {
-        return sle_version_at_least('12-SP4', version_variable => $version_variable)
-          && !check_var($version_variable, '13');
+    if ($version eq '15') {
+        return sle_version_at_least('12-SP3', version_variable => $version_variable)
+          && !check_var($version_variable, '15');
```

Comment:
Now I see the error. It was already wrong for the '13' version. "at least version 15" should be "at least 12-SP4 but not 12-SP4". It does not really matter where the packages are branched of or if SLE 12 SP4 will be developed in parallel to SLE 15. From the customers point of view and our point of view it should be "higher than any 12-something version" =>

```perl
if ($version eq '15') {
    return sle_version_at_least('12-SP4', version_variable => $version_variable)
      && !check_var($version_variable, '12-SP4');
}
```

This will allow to also replace all `check_var('VERSION', '15')` uses.